src/htr2hpc/templates/users/profile.html

```
@@ -14,7 +14,7 @@ <h3>First time setup</h3>
                 <a href="https://informationsecurity.princeton.edu/connecting-to-princeton-n">like so</a>.</li>
             <li>Open Terminal or Command Line on your computer,
                 paste in the below code, and press enter. You will be prompted for your NetID password.<br>
-                <code>ssh {{ request.user.username }}@adroit.princeton.edu 'mkdir -p ~/.ssh &amp;&amp; echo "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIJzoR8jstrofzFKVoiXSFP5jGw/WbXHxFyIaS5b4vSWC test-htr.lib.princeton.edu" >> ~/.ssh/authorized_keys'</code>
+                <code>ssh {{ request.user.username }}@adroit.princeton.edu 'mkdir -p ~/.ssh &amp;&amp; echo "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIJzoR8jstrofzFKVoiXSFP5jGw/WbXHxFyIaS5b4vSWC {{ current_site.domain }}" >> ~/.ssh/authorized_keys'</code>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)